src/main/scala/io/sdkman/changelogs/TaxiMigrations.scala

```diff
@@ -0,0 +1,22 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "069")
+class TaxiMigrations {
+  @ChangeSet(
+    order = "001",
+    id = "001_add_taxi_candidate",
+    author = "martypitt"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "taxi",
+      name = "Taxi",
+      description =
+        "Taxi is a language for semantically describing data and services",
```

Comment:
Can we have a fuller description? This is what is included under `sdk list` and on our website [SDKs](https://sdkman.io/sdks) page.

Reply:
Sure thing -- I've fleshed it out.